OptimizelySDK/OptimizelySDK.csproj

```diff
@@ -86,6 +86,7 @@
     <Compile Include="Entity\ForcedVariation.cs" />
     <Compile Include="Entity\Group.cs" />
     <Compile Include="Entity\IdKeyEntity.cs" />
+    <Compile Include="Entity\OptimizelyJson.cs" />
```

Comment:
This shouldn't be in Entity. It should be at same place where Optimizely is or create a folder and put it there.